sway/sway.5.scd

```diff
@@ -209,6 +209,44 @@ set|plus|minus|toggle <amount>
 	effect on the output the window is currently on. See *sway-output*(5) for
 	further details.
 
+*mirror* start <entire|container|box> <dst_name> ...
+	Begins a mirror session. Multiple sessions may run concurrently.
+	The destination (_dst_) output will be vacated by sway and the mirrored
+	content displayed there.
+	Content will be scaled to fit the width or height of _dst_.
+	See *sway-output*(5) for details on specifying the output name.
+
+*mirror* start _entire_ <dst_name> <src_name> [show_cursor]
+	Mirrors the whole of the source (_src_) output on _dst_. The cursor on
+	_src_ is optionally shown.
+
+*mirror* start _box_ <dst_name> <x> <y> <width> <height> [show_cursor]
```

Comment:
This flavour is of questionable value. If removed, the remaining two could be merged into something like:

`mirror start <src_name|con_id|focussed> <dst_name> [show_cursor]`